packages/l/libwebp/xmake.lua

```diff
@@ -7,38 +7,43 @@ package("libwebp")
     add_urls("https://github.com/webmproject/libwebp/archive/$(version).tar.gz",
              "https://github.com/webmproject/libwebp.git")
     add_versions("v1.1.0", "424faab60a14cb92c2a062733b6977b4cc1e875a6398887c5911b3a1a6c56c51")
+    add_versions("v1.2.0", "d37571723f531e5002667632c94fa18857ed7eb5b0561e3b49e913c3e0b1403e")
+    add_versions("v1.2.1", "7926985218c9e546069c2013dd93774aac3f012fd275247f82b0c119ec9a3801")
     add_versions("v1.2.2", "51e9297aadb7d9eb99129fe0050f53a11fcce38a0848fb2b0389e385ad93695e")
+    add_versions("v1.3.0", "dc9860d3fe06013266c237959e1416b71c63b36f343aae1d65ea9c94832630e1")
 
     add_configs("anim_utils", {description = "Build animation utilities.", default = false, type = "boolean"})
     add_configs("cwebp",      {description = "Build the cwebp command line tool.", default = false, type = "boolean"})
     add_configs("dwebp",      {description = "Build the dwebp command line tool.", default = false, type = "boolean"})
     add_configs("gif2webp",   {description = "Build the gif2webp conversion tool.", default = false, type = "boolean"})
     add_configs("img2webp",   {description = "Build the img2webp animation tool.", default = false, type = "boolean"})
-    add_configs("vwebp",      {description = "Build the vwebp viewer tool.", default = false, type = "boolean"})
+    add_configs("vwebp",      {description = "Build the vwebp viewer tool.", default = flase, type = "boolean"})
     add_configs("webpinfo",   {description = "Build the webpinfo command line tool.", default = false, type = "boolean"})
     add_configs("libwebpmux", {description = "Build the libwebpmux library.", default = false, type = "boolean"})
     add_configs("webpmux",    {description = "Build the webpmux command line tool.", default = false, type = "boolean"})
-    add_configs("sharpyuv",   {description = "Build the sharpyuv library.", default = false, type = "boolean"})
```

Comment:
here

Reply:
Let me see what's going on

Reply:
Found, this option was removed from https://github.com/webmproject/libwebp/commit/d55d447c9a4c7aea0e169a87fea98706a34e897c
I will add this option back for compatibility and leave a deprecation note